### PR TITLE
Test cypress

### LINKED ISSUE #47: 【DevOps】前端 Gtihub CI 建立

### PR DESCRIPTION
**Pull request by Irisloverain**
resolves #47

### CODE REVIEW

File: .github/workflows/e2e-test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: E2E Tests
+
+#測1
+on:
+  pull_request:
+    branches:
+      - test_Cypress
```
Comment (snowmancc) [inline .github/workflows/e2e-test.yml:7]: 這個要改成 main

Comment (Irisloverain) [reply .github/workflows/e2e-test.yml]: 忘記改><感謝!!!

**Review by snowmancc — APPROVED** [review]
LGTM

**Review by timt1028 — APPROVED** [review]
LGTM

**Review by dawnglede — APPROVED** [review]
LGTM

**Pull request merged**